quickwit-proto/proto/search_api.proto

```diff
@@ -167,3 +170,47 @@ message FetchDocsResult {
   // List of complete hits.
   repeated Hit hits = 1;
 }
+
+
+// -- Export -------------------
+
+
+message ExportRequest {
+  // Index ID
+  string index_id = 1;
+
+  // Query
+  string query = 2;
+
+  // Fields to search on
+  repeated string  search_fields = 3;
+  
+  // Time filter
+  optional int64 start_timestamp = 4;
+  optional int64 end_timestamp = 5;
+
+  // Maximum number of hits to return.
+  uint64 max_hits = 6;
```

Comment:
Do we really handle max_hits? That's non trivial.

Reply:
I have removed this, let's add this when we actually make use of it